packages/token-bridge-sdk/src/withdrawals/fetchWithdrawals.ts

```diff
@@ -0,0 +1,243 @@
+import { fetchTokenWithdrawalsFromSubgraph } from './fetchTokenWithdrawalsFromSubgraph'
+import { Provider } from '@ethersproject/providers'
+import { fetchETHWithdrawalsFromSubgraph } from './fetchETHWithdrawalsFromSubgraph'
+import { L2ToL1EventResultPlus } from '../hooks/arbTokenBridge.types'
+
+import { fetchL2BlockNumberFromSubgraph } from '../util/subgraph'
+import { fetchETHWithdrawalsFromEventLogs } from './fetchETHWithdrawalsFromEventLogs'
+import { fetchTokenWithdrawalsFromEventLogs } from './fetchTokenWithdrawalsFromEventLogs'
+import {
+  mapETHWithdrawalToL2ToL1EventResult,
+  mapTokenWithdrawalFromEventLogsToL2ToL1EventResult,
+  mapTokenWithdrawalFromSubgraphToL2ToL1EventResult,
+  tryFetchLatestSubgraphBlockNumber,
+  updateAdditionalWithdrawalData
+} from '../util/withdrawals'
+
+/* Fetch complete withdrawals - both ETH and Token withdrawals from subgraph and event logs into one list */
+/* Also fills in any additional data required per transaction for our UI logic to work well */
+export const fetchWithdrawals = async ({
+  address, // wallet address
```

Comment:
Let's rename the variable directly, so it's more clear? 😄

Reply:
Done